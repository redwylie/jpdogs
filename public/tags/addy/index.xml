<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Addy on JP Dogs</title>
    <link>//localhost:1313/tags/addy/</link>
    <description>Recent content in Addy on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Sun, 05 May 2024 00:00:00 -0400</lastBuildDate><atom:link href="//localhost:1313/tags/addy/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Addy</title>
      <link>//localhost:1313/albums/addy/</link>
      <pubDate>Sun, 05 May 2024 00:00:00 -0400</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/addy/</guid><media:content url="//localhost:1313/albums/addy/Addy-20240505-1_hu8d5b3791234f566d1897a938e3933195_1256440_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/addy/Addy-20240505-1_hu8d5b3791234f566d1897a938e3933195_1256440_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>