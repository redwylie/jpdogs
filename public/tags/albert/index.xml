<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Albert on JP Dogs</title>
    <link>//localhost:1313/tags/albert/</link>
    <description>Recent content in Albert on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Tue, 23 Jul 2024 00:00:00 -0400</lastBuildDate><atom:link href="//localhost:1313/tags/albert/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Albert</title>
      <link>//localhost:1313/albums/albert/</link>
      <pubDate>Tue, 23 Jul 2024 00:00:00 -0400</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/albert/</guid><media:content url="//localhost:1313/albums/albert/Albert-20240723-1_hu122d9a2e0f2646da74100c509a5d53c4_1391099_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/albert/Albert-20240723-1_hu122d9a2e0f2646da74100c509a5d53c4_1391099_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>