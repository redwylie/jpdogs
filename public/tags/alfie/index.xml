<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Alfie on JP Dogs</title>
    <link>//localhost:1313/tags/alfie/</link>
    <description>Recent content in Alfie on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Wed, 29 May 2024 00:00:00 -0400</lastBuildDate><atom:link href="//localhost:1313/tags/alfie/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Alfie</title>
      <link>//localhost:1313/albums/alfie/</link>
      <pubDate>Wed, 29 May 2024 00:00:00 -0400</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/alfie/</guid><media:content url="//localhost:1313/albums/alfie/Alfie-20240529-1_hu375780740319091c4f3d6fc9d17cc701_949173_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/alfie/Alfie-20240529-1_hu375780740319091c4f3d6fc9d17cc701_949173_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>