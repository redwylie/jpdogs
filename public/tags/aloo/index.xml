<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Aloo on JP Dogs</title>
    <link>//localhost:1313/tags/aloo/</link>
    <description>Recent content in Aloo on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Wed, 29 May 2024 00:00:00 -0400</lastBuildDate><atom:link href="//localhost:1313/tags/aloo/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Aloo</title>
      <link>//localhost:1313/albums/alo/</link>
      <pubDate>Wed, 29 May 2024 00:00:00 -0400</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/alo/</guid><media:content url="//localhost:1313/albums/alo/Aloo-20240327-1_hubd59fc9686f9fa469c46c1ffc9e0db20_1706274_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/alo/Aloo-20240327-1_hubd59fc9686f9fa469c46c1ffc9e0db20_1706274_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>