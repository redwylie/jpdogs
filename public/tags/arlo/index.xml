<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Arlo on JP Dogs</title>
    <link>//localhost:1313/tags/arlo/</link>
    <description>Recent content in Arlo on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Mon, 01 Jan 2024 00:00:00 -0500</lastBuildDate><atom:link href="//localhost:1313/tags/arlo/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Arlo</title>
      <link>//localhost:1313/albums/arlo-2/</link>
      <pubDate>Mon, 01 Jan 2024 00:00:00 -0500</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/arlo-2/</guid><media:content url="//localhost:1313/albums/arlo-2/Arlo-20240608-1_hue5a4f482216397a4acb988a155f8bb59_955354_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/arlo-2/Arlo-20240608-1_hue5a4f482216397a4acb988a155f8bb59_955354_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
    <item>
      <title>Arlo</title>
      <link>//localhost:1313/albums/arlo-3/</link>
      <pubDate>Mon, 01 Jan 2024 00:00:00 -0500</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/arlo-3/</guid><media:content url="//localhost:1313/albums/arlo-3/Arlo-20240708-1_hu2bcb0762f6ee1ffe1dbed93af4229cfc_1146195_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/arlo-3/Arlo-20240708-1_hu2bcb0762f6ee1ffe1dbed93af4229cfc_1146195_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
    <item>
      <title>Arlo</title>
      <link>//localhost:1313/albums/arlo/</link>
      <pubDate>Mon, 01 Jan 2024 00:00:00 -0500</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/arlo/</guid><media:content url="//localhost:1313/albums/arlo/Arlo-20240508-1_huf7108528b80e4f26c0dba1eee288e080_1344559_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/arlo/Arlo-20240508-1_huf7108528b80e4f26c0dba1eee288e080_1344559_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>