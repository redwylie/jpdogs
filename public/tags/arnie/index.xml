<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Arnie on JP Dogs</title>
    <link>//localhost:1313/tags/arnie/</link>
    <description>Recent content in Arnie on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright>
    <lastBuildDate>Mon, 01 Jan 2024 00:00:00 -0500</lastBuildDate><atom:link href="//localhost:1313/tags/arnie/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Arnie</title>
      <link>//localhost:1313/albums/arnie/</link>
      <pubDate>Mon, 01 Jan 2024 00:00:00 -0500</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/albums/arnie/</guid><media:content url="//localhost:1313/albums/arnie/Arnie-20240613-1_hu3c2baedb88b99c8b582132b39969ee42_1524867_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/albums/arnie/Arnie-20240613-1_hu3c2baedb88b99c8b582132b39969ee42_1524867_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;&lt;/p&gt;
        </description></item>
  </channel>
</rss>