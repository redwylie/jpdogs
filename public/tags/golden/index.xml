<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Golden on JP Dogs</title>
    <link>//localhost:1313/tags/golden/</link>
    <description>Recent content in Golden on JP Dogs</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <managingEditor>redwylie@gmail.com (Mark Wylie)</managingEditor>
    <webMaster>redwylie@gmail.com (Mark Wylie)</webMaster>
    <copyright>Â© Mark Wylie</copyright><atom:link href="//localhost:1313/tags/golden/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Sophie</title>
      <link>//localhost:1313/sophie/</link>
      <pubDate>Mon, 01 Jan 0001 00:00:00 +0000</pubDate><author>redwylie@gmail.com (Mark Wylie)</author>
      <guid>//localhost:1313/sophie/</guid><media:content url="//localhost:1313/sophie/Sophie-20240416-1_hu274090381d44567846421c5670f1e525_840087_900x600_fill_q95_box_smart1.jpg" type="image/jpeg"/>
        <description>
          &lt;img src=&quot;//localhost:1313/sophie/Sophie-20240416-1_hu274090381d44567846421c5670f1e525_840087_900x600_fill_q95_box_smart1.jpg&quot; /&gt;
          &lt;p&gt;Sophie&lt;/p&gt;
        </description></item>
  </channel>
</rss>